docs/content/providers/docker.md

```diff
@@ -12,8 +12,7 @@ A Story of Labels & Containers
 
 Attach labels to your containers and let Traefik do the rest!
 
-Traefik works with both [Docker (standalone) Engine](https://docs.docker.com/engine/)
-and [Docker Swarm Mode](https://docs.docker.com/engine/swarm/).
+Traefik works with [Docker (standalone) Engine](https://docs.docker.com/engine/).
```

Comment:
I don't know exactly how to reword this if so, but it seems that previously the intention was just to yield about the two modes, so do we need it anymore?
Maybe keeping the previous intention by adding a link to the swarm provider documentation page could make sense.